sdk/pom.xml

```diff
@@ -686,6 +686,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
We should probably discuss whether we want to bring in a dependency on guava.testlib. If we do, it should probably be in `<scope>test</scope>`.